docs/ruleset_functions/get_heating_fuels_p.md

```diff
@@ -0,0 +1,75 @@
+# get_heating_fuels_p
+
+**Description:** Get a list of the heating fuels used in the P-RMR.  
```

Comment:
I would make it agnostic to the type of the RMR, so that it works with either U, P or B-RMR.

Reply:
Good point